### ISSUE TITLE
migrate the app from ACS to AAD Auth

### ISSUE THREAD

**Issue by moelazha** [Azure-Samples/media-services-dotnet-live-monitoring-dashboard#4]
Just wanted to point on this.
Since ACS is deprecated, the app would need to be updated to AAD auth (and some screenshots from the new portal)

**Comment by Kofl** [comment]
Indeed, really frustrating didn't get it to work with the new portal and gave up after 2 days.
